content/applications/sales/sales/products_prices/products/variants.rst

```diff
@@ -105,14 +77,21 @@ The :guilabel:`Display Type` options are:
 - :guilabel:`Select`: options appear in a drop-down menu on the product page of the online store.
 - :guilabel:`Color`: options appear as small, colored squares, which reflect any HTML color codes
   set, on the product page of the online store.
+- :guilabel:`Multi-checkbox (option)`: options appear as selectable checkboxes on the product page
+  of the online store.
```

Comment:
Nitpick suggestion to change the order of these display types so they match the order of the UI 😄
```suggestion
- :guilabel:`Pills`: options appear as selectable buttons on the product page of the online store.
- :guilabel:`Color`: options appear as small, colored squares, which reflect any HTML color codes
- :guilabel:`Radio`: options appear in a bullet-style list on the product page of the online store.
- :guilabel:`Select`: options appear in a drop-down menu on the product page of the online store.
  set, on the product page of the online store.
- :guilabel:`Multi-checkbox (option)`: options appear as selectable checkboxes on the product page
  of the online store.
```